utils/S3UtilsMongoClient.js

```diff
@@ -150,6 +151,7 @@ class S3UtilsMongoClient extends MongoClientInterface {
                             entry: res,
                             error,
                         });
+                        monitoring.objectsCount.inc({ state: 'error' });
```

Comment:
seems like the label is not a state of the object : more like `result` or `status`

Reply:
changed to `status` : https://github.com/scality/s3utils/pull/312/commits/be0f3e16b518690540561a94a498ff07dc3860b0#diff-612a4c10e405510cdaad879589d4a7d157df6e0ff42b4645d2a21d4d17e41fa9R30-R34